src/Teapot.Web/Web.config

```diff
@@ -1,37 +1,13 @@
-﻿<?xml version="1.0"?>
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Can't we fully ditch the web.config?

Reply:
We can, yes.

The idea was to remove headers that expose information about the stack that is used.
That being said, the repo being public sort of defeats the point.
I'll remove it.